Tests/AnyLintTests/Checkers/TemplateCheckerTests.swift

```diff
@@ -0,0 +1,59 @@
+@testable import AnyLint
+@testable import Utility
+import XCTest
+
+final class TemplateCheckerTests: XCTestCase {
+    override func setUp() {
+        log = Logger(outputType: .test)
+        TestHelper.shared.reset()
+    }
+
+    func testPerformWithLocalSource() {
+        withTemporaryFiles([(subpath: "AnyLint/sample.swift", contents: "")]) { filePaths in
+            _ = try! TemplateChecker(source: .local(filePaths[0]), runOnly: nil, exclude: nil, logDebugLevel: false).performCheck()
+            XCTAssertEqual(TestHelper.shared.consoleOutputs.count, 1)
+            XCTAssertEqual(TestHelper.shared.consoleOutputs[0].message, "Local template file to run: '\(filePaths[0])'")
+            XCTAssertEqual(TestHelper.shared.consoleOutputs[0].level, .info)
+        }
+    }
+
+    func testPerformWithRemoteSource() {
+        _ = try! TemplateChecker(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Force tries should be avoided.](https://app.codacy.com/gh/Flinesoft/AnyLint/pullRequest?prid=5691372)